### PR TITLE
[WHO-60] Add static intro about carousel and menu page

### PR DESCRIPTION
**Pull request by myire**

### CODE REVIEW

File: client/ionic/src/pages/About.module.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.page ion-header ion-toolbar {
+  --background: translucent;
+  --background: blue;
```
Comment (advayDev1) [inline client/ionic/src/pages/About.module.css]: not sure what the effect of double specifying this property is?  does it mean anything?

File: client/ionic/src/pages/Menu.module.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.page ion-header ion-toolbar {
+  --background: translucent;
+  --background: blue;
```
Comment (advayDev1) [inline client/ionic/src/pages/Menu.module.css]: ditto

Comment (myiremark) [reply client/ionic/src/pages/Menu.module.css]: removed in https://github.com/WorldHealthOrganization/app/pull/74

File: client/ionic/src/pages/About.module.css (hunk 1 above)
Comment (myiremark) [reply client/ionic/src/pages/About.module.css]: removed in https://github.com/WorldHealthOrganization/app/pull/74

**Comment by myiremark** [comment]
Created screenshots, we need to figure out where to put them and are we adding to docs.

Closing this PR in favor of https://github.com/WorldHealthOrganization/app/pull/74 

also, how do we want to handle merges of upstream?  That may just have been from me being an idiot last night

**Pull request closed without merging**